### DISCUSSION TITLE
Set a different background color for the searched genomic coordinates

### DISCUSSION THREAD

**Discussion by LankyAlec** [GMOD/jbrowse-components#4467] [Ideas]
Hi,

Several users of the genome browser, version react-linear-genome-view, which I manage, are complaining that when they set genomic coordinates and then modify them to look at the surrounding genes by moving the window or adjusting the zoom, they lose track of the original coordinates.

How could this be avoided? Could we perhaps use a different background color for the original genomic coordinates? Or should we consider another solution?

Thank you

**Comment by cmdcolin** [comment]
the workflow you are asking about is that they use the in-app search box, and then this navigates somewhere, and they want that region where they navigated to be highlighted?

we could consider making a system to do something like this by default

**Reply by cmdcolin** [reply]
another option would be installing @jbrowse/plugin-grid-bookmark (https://www.npmjs.com/package/@jbrowse/plugin-grid-bookmark) and then this allows users to bookmark regions manually, or writing a custom plugin that does it

**Comment by cmdcolin** [comment]
i made an issue here https://github.com/GMOD/jbrowse-components/issues/4486 for this just to keep track of it, i don't have any  updates on it though

**Comment by cmdcolin** [comment]
long time coming but our work on PR #5468 now fixed this. looking for beta testers if interested
